pxtwinrt/hiddeploy.ts

```diff
@@ -49,44 +53,78 @@ namespace pxt.winrt {
 
         initAsync(): Promise<void> {
             Util.assert(!this.dev, "HID interface not properly reseted");
-            const selector = Windows.Devices.HumanInterfaceDevice.HidDevice.getDeviceSelector(0xFF97, 0x0001);
-            return pxt.winrt.promisify(Windows.Devices.Enumeration.DeviceInformation.findAllAsync(selector, null)
-                .then(devices => {
-                    pxt.debug(`hid enumerate ${devices.length} devices`)
+            const wd = Windows.Devices;
+            const whid = wd.HumanInterfaceDevice.HidDevice;
+            if (!WindowsRuntimeIO.uf2Selectors) {
+                this.initUf2Selectors();
+            }
+            const getDevicesPromise = WindowsRuntimeIO.uf2Selectors.reduce((soFar, currentSelector) => {
+                // Try all selectors, in order, until some devices are found
+                return soFar.then((devices) => {
```

Comment:
Does the new code try to query all the HID devices matching the selectors? If yes can we postpone that change until we test that scenario. Old code seem to query just the first device.